recipes-connectivity/prplmesh/prplmesh.bb

```diff
@@ -8,27 +8,123 @@ inherit cmake pythonnative
 S = "${WORKDIR}/git"
 OECMAKE_SOURCEPATH = "${S}/"
 
+SUMMARY = "prplMesh"
 SECTION = "net"
 LICENSE = "BSD-2-Clause-Patent.txt"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
-DEPENDS = "python-native python-pyyaml-native json-c openssl zeromq"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=0518d409dae93098cca8dfa932f3ab1b"
+
+DEPENDS = "python-native python-pyyaml-native json-c openssl"
 RDEPENDS_${PN} = "iproute2 busybox readline"
-SUMMARY = "prplMesh"
 
-SRCREV = "911392026280d9961823a9359e6cc4f08c9ee0a2"
-SRC_URI = "git://github.com/prplfoundation/prplMesh.git;protocol=http;branch=master"
+SRCREV = "8a331d9fa1631782cf06603ee612ce6133ee0c1a"
+SRC_URI = "git://github.com/prplfoundation/prplMesh.git;protocol=http;branch=master \
+           file://001-add_find_nl_for_nl80211_bwl_type.patch \
+           file://002-allow_redefine_beerocks_vars_from_cmdline.patch \
+           file://003_prplmesh_util_stop_use_hard_coded_wlan_iface_names.patch \
+           file://004-get_hostapd_ctrl_iface_pathes_from_conf.patch"
+
 
-BUILD_TYPE ?= "Release"
+PACKAGECONFIG ?= "${BWL_TYPE} ${MSGLIB} ${BUILD_TYPE}"
+
+BUILD_TYPE ?= "release"
 TARGET_PLATFORM ?= "linux"
-BWL_TYPE ?= "DUMMY"
+MSGLIB ?= "mq-zmq"
+BWL_TYPE ?= "bwl-nl80211"
 INSTALL_PREFIX ?= "/opt/prplmesh"
+#
+# QEMU MACHINE specific
```

Comment:
I think we need to add a condition, which will check DISTRO_FEATURES and enable variables for qemu if qemu present in DISTRO_FEATURES.

Reply:
Explicit condition is not needed.
I'm using override syntax with MACHINE name (qemux86, qemuarm, etc.)
As a result variables with such suffix (_qemux86, etc.) will be used only if `${MACHINE}` is equal to `qemux86`. etc.